Source/Blazorise/Components/Table/TableHeaderCell.razor.cs

```diff
@@ -140,6 +140,15 @@ protected override ValueTask DisposeAsync( bool disposing )
 
     #region Properties
 
+    /// <summary>
+    /// Gets the fixed position data attribute.
+    /// </summary>
+    protected string FixedPositionDataAttribute => FixedPosition == TableColumnFixedPosition.Start
```

Comment:
Why the `start` vs `end` matters? the js seems to care just about the thing not being null.

Reply:
This is for the future sake. And in case we, or users would like to know that information. So it is better to have full info since we already need this attribute.

Reply:
ok, ok.

then, why not switch expression instead the double conditional operator??

```
   protected string FixedPositionDataAttribute => FixedPosition switch
    {
        TableColumnFixedPosition.Start => "start",
        TableColumnFixedPosition.End => "end",
        _ => null
    };
```